test/issues/0519.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var assert = require('assert');
+var yaml   = require('../../');
+
+test('Loader should not strip quotes for lines containing equals sign', function () {
+  var line_with_equals_sign = "'='\n";
+  var result_of_load_and_dump = yaml.dump(yaml.load(line_with_equals_sign));
+  assert.strictEqual(line_with_equals_sign, result_of_load_and_dump);
+});
```

Comment:
IMO actual test content could be:

```js
// pyyaml fails with unquoted `=`
// https://yaml-online-parser.appspot.com/?yaml=%3D%0A&type=json
assert.strictEqual(yaml.load(yaml.dump('=')), '=');
assert.strictEqual(yaml.dump('='), "'='");
```

Reply:
Thanks for the suggestion! I updated the test.